src/ipc/cancel-speak.ts

```diff
@@ -0,0 +1,26 @@
+import makeDebug from 'debug';
+import { IpcMain } from 'electron';
+import promisifiedExec from '../utils/promisifiedExec';
```

Comment:
As per discussion in Slack, this `promisifiedExec` utility is not safe to use, let's remove it and use `utils/exec.ts` instead.